SCERES_env.yml

```diff
@@ -0,0 +1,138 @@
+name: SCERES
+channels:
+  - bioconda
+  - anaconda
+  - conda-forge
+  - defaults
+dependencies:
+  - _libgcc_mutex=0.1=conda_forge
+  - _openmp_mutex=4.5=1_gnu
+  - anyio=2.0.2=py38h578d9bd_4
+  - argon2-cffi=20.1.0=py38h497a2fe_2
+  - async_generator=1.10=py_0
+  - attrs=20.3.0=pyhd3deb0d_0
+  - babel=2.9.0=pyhd3deb0d_0
+  - backcall=0.2.0=pyh9f0ad1d_0
+  - backports=1.0=py_2
```

Comment:
- Could you rename file `SCERES_env.yml` in `environment.yml`? This is a convention.
- Could you rename environment name in lower case (`sceres`)?
- There are too much details in the file. Could you add only packages you manually installed?
- Can you remove the `anaconda` channel?

Reply:
Can you replace the `dependencies` section with this one:
```
# channels priority
# see https://bioconda.github.io/
# conda-forge has highest prioriy
channels:
    - defaults
    - bioconda
    - conda-forge
```